packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -127,6 +179,26 @@ export class KubernetesClient {
     });
   }
 
+  setupKubeWatcher() {
+    this.kubeWatcher?.abort();
+    const ns = this.currrentNamespace;
+    if (ns) {
+      new Watch(this.kubeConfig)
+        .watch(
+          '/api/v1/namespaces/' + ns + '/pods',
+          {},
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          (phase: string, apiObj: any, watchObj?: any) => {
+            this.apiSender.send('pod-event');
+          },
+          (err: any) => {
+            console.log('Kube event error ' + err);
```

Comment:
```suggestion
            console.error('Kube event error', err);
```

should it be reported to a client like with `pod-event-error` or we don't care ?